syntax = "proto3";

package rpc;

message MessageHeader
{
    uint32 request_size=1;
    uint32 arg_size=3;
}
